### PR TITLE
add my readme

### PR DESCRIPTION
**Pull request by sudip-mondal-2002**

### CODE REVIEW

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:1]:
```suggestion
# Sudip Mondal

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:5]:
```suggestion
## Skills

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:116]:
```suggestion
### Personal Projects

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:138]:
```suggestion
### Group projects

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:150]:
```suggestion
### Project under Professors

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:157]:
```suggestion
## Professional Experiences

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:170]:
```suggestion
## Position of Responsibilities

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:175]:
```suggestion
## Co-curricular Activities

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:180]:
```suggestion
## Scholastic Achievements

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Awesome readme! 🚀
These are all some suggestions by the Markdown linter. 🙂

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:5]:
```suggestion
[E-mail](mailto:sudipmondal.2002@rediffmail.com) | [LinkedIn](https://www.linkedin.com/in/sudip-mondal-374a12201/) | [Facebook](https://www.facebook.com/sudip16022002/) | [Instagram](https://www.instagram.com/sudip._.mondal/) | [Twitter](https://twitter.com/SUDIPMO19931261) | [YouTube](https://www.youtube.com/channel/UCvfY9VkXaFNxQJO7lfDV6Vw) | [GitHub](https://github.com/sudip-mondal-2002) | [Website](https://sudipmondal.tk) | [NPM](https://www.npmjs.com/~sudipmondal) | [PyPI](https://pypi.org/user/sudip-mondal-2002/)

```

File: Readmes/Sudip Mondal/README.md
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:176]:
```suggestion
   * Worked as Subject Matter Expert for mathematics.

```

**Review by Panquesito7 — APPROVED** [review]
LGTM! 🚀

**Comment by sudip-mondal-2002** [comment]
How to get it merged?

**Comment by Panquesito7** [comment]
> How to get it merged?

A maintainer has to merge it. I'm not a maintainer, so I can't merge it. 😅

**Review by DevrajDC — APPROVED** [review]
Looks good, merging it 🚀

**Pull request merged**
